lisk-dex-electron/package.json

```diff
@@ -15,7 +15,8 @@
     "react-dom": "^16.13.1",
     "react-scripts": "^3.4.1",
     "recharts": "^1.8.5",
-    "runtypes": "^4.1.0"
+    "runtypes": "^4.1.0",
+    "sodium-native": "^3.2.1"
```

Comment:
What is the reason for importing sodium-native here?
It's not a direct dependency of lisk-dex-electron. It's a dependency of one of the blockchain clients.

Reply:
I couldn't run `npm i` without `sodium-native` it fails, I added it but can take it out if needed.

Reply:
@jondubois I removed `sodium-native` from the `package.json`